### ISSUE TITLE
Send e-mail to advertiser and highest bidder if auction is over

### ISSUE THREAD

**Issue by Dean442** [scg-unibe-ch/ese2016-team8#14]
The owner as well as the highest bidder should be contacted if the auction is over.

**Comment by Dean442** [comment]
60a82de1b0e2835dbb43aa8795d8ebdd9db3ff38 they get contacted by whoever user visits the site after the auction is over. However the boolean hasEnded on the ad doesn't get updated so far, hence every time some user visits the page the e-mails will be sent again.

**Comment by remoroethlisberger** [comment]
[60bdba3](https://github.com/scg-unibe-ch/ese2016-team8/commit/60bdba3c5dfd023622bc7b123f6c039e5e2e01e7) "Complete" overhaul of the auction implementation, so that it makes much more sense! It is still implemented as an Ad, but the ending of the auction is now stored properly in the correct field. This means, that we need to change all the c:if stuff. Since the path changed from "MoveOutDate" to "AuctionEndDate"... So there is a lot of work ahead, but it makes it much easier to send the Notification. Thanks to @Dean442, who brought me on this idea, we can send message when loading the pages...
This happens now in the AdService class, where I implemented a short and easy way to send the message. This message will only be sent once! It also makes more sense to have the functionality inside the AdService, so that we won't have to much of a mess in our jsp files! ;)

Since there will be a lot of changes, I decided to open up a new branch [auctionEndingAndNotification](https://github.com/scg-unibe-ch/ese2016-team8/tree/auctionEndingAndNotification)
